Plugins/BTCPayServer.Plugins.Blink/BlinkLightningClient.cs

```diff
@@ -298,32 +299,48 @@ public async Task<LightningPayment[]> ListPayments(CancellationToken cancellatio
     public async Task<LightningInvoice> CreateInvoice(CreateInvoiceParams createInvoiceRequest,
         CancellationToken cancellation = new())
     {
-        var reques = new GraphQLRequest
-        {
-            Query = @"
-mutation LnInvoiceCreate($input: LnInvoiceCreateInput!) {
-  lnInvoiceCreate(input: $input) {
+        string btcLnInvoiceCreate = @"
+mutation LnInvoiceCreate($input: LnInvoiceCreateOnBehalfOfRecipientInput!) {
+  lnInvoiceCreateOnBehalfOfRecipient(input: $input) {
     invoice {
       createdAt
       paymentHash
       paymentRequest
       paymentSecret
       paymentStatus
       satoshis
-      
     }
   }
-}",
+}";
+
+        string usdLnInvoiceCreate = @"
+mutation LnUsdInvoiceCreate($input: LnUsdInvoiceBtcDenominatedCreateOnBehalfOfRecipientInput!) {
```

Comment:
We have a different mutation for usd invoice creation (if the walletId is for a stablesats wallet)